adapters/repos/db/docid/scan.go

```diff
@@ -77,7 +77,23 @@ func (os *objectScannerLSM) init() error {
 }
 
 func (os *objectScannerLSM) scan() error {
+	// each object is scanned one after the other, so we can reuse the same memory allocations for all objects
 	docIDBytes := make([]byte, 8)
+
+	// Preallocate strings needed for json unmarshalling
+	var propStrings [][]string
+	for _, s := range os.properties {
+		propStrings = append(propStrings, []string{s})
+	}
```

Comment:
Do you think that it would be beneficial for us also to pre-allocate `propStrings` array, like this?
```
propStrings := make([][]string, len(os.properties))
for i := range os.properties {
	propStrings[i] = []string{os.properties[i]}
}
```

Reply:
changed it, but will not make a difference. This is only called once per aggregation and usually only has a few strings (unless you have monster-classes with 10000properties)